packages/cli/src/presets/codemods/codemods.tsx

```diff
@@ -33,6 +33,19 @@ const codemodChoice: Array<Choice<keyof CodemodOptions>> = [
     name: 'ignorePattern',
     message: '--ignore-pattern',
   },
+  {
+    name: 'plugins',
+    message: 'number of plugins',
```

Comment:
It's a bit weird you have to specify the number of plugins here, is there no way of specifying the plugin paths in a more straightforward way?

Reply:
yeah, I had a lot of trouble coming up with a nice UX for this with the enquirer library. I may be able to do this with newlines - I'll have another go at it

Reply:
Ended up going to a following prompt with the `List` import